tests/ui/manual_range_patterns.rs

```diff
@@ -45,4 +45,11 @@ fn main() {
         };
     }
     mac!(f);
+
+    let _ = match f {
+        2..=15 => 4,
```

Comment:
shouldn't this use the stylistic OR?

Reply:
Ah, didn't pay attention but it was removed by fmt before I pushed, so suppose that if fmt is removed it doesn't make sense to fix

Reply:
No, it does make sense to fix, write the test with `#[rustfmt::skip]`!

Reply:
Poeple don't always use the same formatter settings

Reply:
Ok, done